node-proxy/test/wsapp.js

```diff
@@ -55,10 +55,12 @@ createWebSocketServer(app8080);
 
 var  sslcerts_path = "../sslcerts/";
 var  server_name = "localhost";
+var  cipher_list = "kEECDH:+kEECDH+SHA:kEDH:+kEDH+SHA:+kEDH+CAMELLIA:kECDH:+kECDH+SHA:kRSA:" + 
+               "+kRSA+SHA:+kRSA+CAMELLIA:!aNULL:!eNULL:!SSLv2:!RC4:!DES:!EXP:!SEED:!IDEA:+3DES"
```

Comment:
Missing semicolon.  It's optional in JavaScript, but we might as well include it for consistency.